### DISCUSSION TITLE
ERR_TOO_MANY_REDIRECTS on new domain

### DISCUSSION THREAD

**Discussion by BarishSarac** [vercel/community#4698] [Help] [answered]
### Summary

Hello, I added my website to vercel and configured DNS hours ago but still get `ERR_TOO_MANY_REDIRECTS` error. Does it take long to propagate?

### Steps to Reproduce

go to rocote.com

**Comment by amyegan** [comment, marked as answer]
If you're using Cloudflare, this guide should help: https://vercel.com/guides/resolve-err-too-many-redirects-when-using-cloudflare-proxy-with-vercel

There's also a troubleshooting guide in our Domains documentation with some other ideas to try: https://vercel.com/docs/projects/domains/troubleshooting

**Reply by haifzhan** [reply]
Thanks for sharing, it helps!
